config/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+        "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+
+<module name="Checker">
+    <property name="charset" value="UTF-8" />
+
+    <!-- Checks Java files and forbids empty Javadoc comments -->
+    <module name="RegexpMultiline">
+        <property name="id" value="EmptyJavadoc"/>
+        <property name="format" value="\/\*[\s\*]*\*\/"/>
+        <property name="fileExtensions" value="java"/>
+        <property name="message" value="Empty javadoc comments are forbidden"/>
+    </module>
+
+    <module name="TreeWalker">
+        <module name="AvoidStarImport" />
+
+        <!-- Unused imports are forbidden -->
+        <module name="UnusedImports" />
```

Comment:
Doesn't spotless remove unused imports?

Reply:
Yes, Spotless does. But, since Spotless can't check as many rules as Checkstyle, I wanted to be sure that Checkstyle had the full list of rules. This way there is one place to see the currently enforced rules.